tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -80,30 +81,22 @@ size_t N = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
 size_t J_CL_DEVICE_MAX_WORK_GROUP_SIZE0 = J_CL_DEVICE_CONST_WORK_GROUP_SIZE0;
 size_t J_CL_DEVICE_MAX_WORK_GROUP_SIZE1 = J_CL_DEVICE_CONST_WORK_GROUP_SIZE1;
 
-struct op_time {
-  struct timeval start;
-  struct timeval end;
-};
+struct counters
+{
+    using clock_type = typename std::chrono::high_resolution_clock;
+    using duration_type = typename std::chrono::milliseconds;
```

Comment:
My question is not about the `using` but the `typename`.
Can this be simplified to:
```
using clock_type = std::chrono::high_resolution_clock;
using duration_type = std::chrono::milliseconds;
```
?